automation/test.sh

```diff
@@ -280,22 +280,28 @@ spec:
     path: /
   storageClassName: windows
 EOF
-  # Run only Windows tests
-  export KUBEVIRT_E2E_FOCUS=Windows
-elif [[ $TARGET =~ (cnao|multus) ]]; then
-  export KUBEVIRT_E2E_FOCUS="Multus|Networking|VMIlifecycle|Expose|Macvtap"
-elif [[ $TARGET =~ sriov.* ]]; then
-  export KUBEVIRT_E2E_FOCUS=SRIOV
-elif [[ $TARGET =~ gpu.* ]]; then
-  export KUBEVIRT_E2E_FOCUS=GPU
-elif [[ $TARGET =~ (okd|ocp).* ]]; then
-  export KUBEVIRT_E2E_SKIP="SRIOV|GPU"
-else
-  export KUBEVIRT_E2E_SKIP="Multus|SRIOV|GPU|Macvtap"
 fi
 
-if [[ "$KUBEVIRT_STORAGE" == "rook-ceph" ]]; then
-  export KUBEVIRT_E2E_FOCUS=rook-ceph
+# Set KUBEVIRT_E2E_FOCUS and KUBEVIRT_E2E_SKIP if KUBEVIRT_E2E_FOCUS not already set
+if [[ -z ${KUBEVIRT_E2E_FOCUS} ]]; then
```

Comment:
I guess that is just a matter of taste, but would it make sense to just make this the first entry in the existing if/else instead of wrapping the existing one? Basically

```
if [[ -n "${KUBEVIRT_E2E_FOCUS}" ]]; then
    # Don't override
 elif [[ ... ]]]
 ```

Reply:
Further, would it make sense to test check for both at the same time, to avoid weird mixtures?

Something like

```
if [[ -n "${KUBEVIRT_E2E_FOCUS}" || -n "${KUBEVIRT_E2E_SKIP}"  ]]; then
    # Don't override
 elif [[ ... ]]]
```

Basically, if one of the two is set, assume full control outside.